user/deployment/script.md

```diff
@@ -53,11 +53,11 @@ deploy:
     all_branches: true
 ```
 
-## Deployment is executed by Ruby 1.9.3
+## Deployment is executed by Ruby 2.2.0
```

Comment:
This is not exactly correct. The correct answer is "depends". Sorry.

This depends on the build environment. It would be quite confusing to explain *what* version is used in which, so I suggest using a different phrase, for example:

> Deployment may be executed by a Ruby runtime different from the one you used for the build.